ui/apps/platform/cypress/integration/vulnerabilities/snoozeWorkflow.test.ts

```diff
@@ -0,0 +1,58 @@
+import withAuth from '../../helpers/basicAuth';
+import { hasFeatureFlag } from '../../helpers/features';
+
+// Note: each test case should be run against both Node and Platform CVE overview pages
+describe('Node and Platform CVEs - Snooze workflow', () => {
+    withAuth();
+
+    before(function () {
+        if (!hasFeatureFlag('ROX_VULN_MGMT_NODE_PLATFORM_CVES')) {
+            this.skip();
+        }
+    });
+
+    it('should restrict the snooze UI controls', () => {
```

Comment:
Similar to comment about following links, it seems most reliable if each permissions scenario has its own test.